src/main/java/edu/stanford/protege/search/lucene/tab/engine/BasicQuery.java

```diff
@@ -178,11 +181,31 @@ private static BooleanQuery createContainsWordQuery(OWLProperty property, String
             return builder.build();
         }
 
-        private static BooleanQuery createContainsPhraseQuery(OWLProperty property, String searchString) {
-            BooleanQuery.Builder builder = new BooleanQuery.Builder();
-            builder.add(LuceneUtils.createTermQuery(IndexField.ANNOTATION_IRI, property.getIRI().toString()), Occur.MUST);
-            builder.add(LuceneUtils.createPhraseQuery(IndexField.ANNOTATION_TEXT, searchString), Occur.MUST);
-            return builder.build();
+        private static BooleanQuery createContainsPhraseQuery(OWLProperty property, String searchString, Analyzer anal) {        	
+
+        	BooleanQuery.Builder builder = new BooleanQuery.Builder();
+        	// split the phrase into words. When a user types in a phrase for the
+        	// value of a basic query there is an implicit conjunction. After splitting the phrase
+        	// each one is parsed to normalize and remove stop words
+        	String[] terms = searchString.split("\\s+");
+        	for (int i = 0; i < terms.length; i++) {
+        		Query qp = null;
+        		try {
+        			QueryParser parser = new QueryParser(IndexField.ANNOTATION_TEXT, anal);
+        			qp = parser.parse(terms[i].replaceAll("\\p{P}", ""));
+        		}
+        		catch (ParseException e) {
+        			qp = LuceneUtils.createTermQuery(IndexField.ANNOTATION_TEXT, ""); // return an empty term query
```

Comment:
This one I'm not so sure about. This snippet is used elsewhere and if it fails to parse 0 results are returned. The same is true if you use the entire phrase, because the indexer will also have failed to parse. I tried both ways and get the same results.